test/profiler/test_profiler_tree.py

```diff
@@ -249,6 +249,7 @@ def assertTreesMatch(self, actual: str, expected: str, allow_failure: bool = Fal
                 else:
                     raise
 
+    @skipIfRocm
```

Comment:
Skipping for now, proper fix to be done in https://github.com/ROCm/frameworks-internal/issues/8556